librz/type/base.c

```diff
@@ -277,6 +282,69 @@ RZ_API RzBaseType *rz_type_db_get_base_type(RzTypeDB *typedb, const char *name)
 	return base_type;
 }
 
+/**
+ * \brief Returns the list of all basic types of the chosen kind
+ *
+ * \param typedb Types Database instance
+ * \param kind Kind of the types to list
+ */
+RZ_API RZ_OWN RzList /* RzBaseType */ *rz_type_db_get_base_types_kind(RzTypeDB *typedb, RzBaseTypeKind kind) {
```

Comment:
Would it make sense to have a `ANY` kind and just use one API instead of two that do the same thing?

Reply:
It wouldn't make sense to add in `RzBaseTypeKind`.

Reply:
Also note, that the bodies of these functions will change when I migrate types storage to the hashtable.

Reply:
agree with @XVilka 
I would call this function `rz_type_db_get_base_types_of_kind()` though, so it's a bit more descriptive.

Reply:
Renamed the function.